includes/Shopping/Shopify_Query.php

```diff
@@ -231,9 +231,31 @@ protected function fetch_remote_products( string $search_term ) {
 		 */
```

Comment:
```
if ( is_wp_error( $body ) ) {
    $body->add_data( [ 'status' => 400 ] );
	return $body;
}
```

We may want to add a status code here or even create our own WP_Error, so we can display a message.